base/src/main/java/com/smartdevicelink/proxy/interfaces/ISdl.java

```diff
@@ -216,8 +217,18 @@
      * @param scListener listener that will be called when the system capability is retrieved. If already cached, it
      *                   will be called immediately
      */
+    @Deprecated
```

Comment:
Please add javadoc as to what method should be used instead of this one because of deprecation.